README.md

```diff
@@ -1,49 +1,55 @@
-# nextstrain.org/tb
+# Nextstrain repository for *Mycobacterium tuberculosis*
+This workflow performs the following analysis:
 
-[![Build Status](https://github.com/nextstrain/tb/actions/workflows/ci.yaml/badge.svg?branch=master)](https://github.com/nextstrain/tb/actions/workflows/ci.yaml)
+* Fetches metadata for *M. tuberculosis* samples with Illumina shotgun sequence data from [NCBI SRA](https://www.ncbi.nlm.nih.gov/sra) using [DuckDB CLI](https://duckdb.org/docs/stable/clients/cli/overview.html)
+* Subsamples the metadata across time and geography
+* Downloads fastq files for subsampled metadata from NCBI SRA using [fasterq-dump](https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump)
+* Assigns lineages and identifies drug resistance variants for each sample using [tb-profiler](https://github.com/jodyphelan/TBProfiler)
+* Creates a multi-sample fasta alignment using [snippy](https://github.com/tseemann/snippy) with low-confidence regions masked following [Marin et al. 2022](https://academic.oup.com/bioinformatics/article/38/7/1781/6502279)
+* Creates a multi-sample VCF of informative sites using a custom script
+* Performs phylogenetic reconstruction using [IQTREE](http://www.iqtree.org/)
 
-This is the [Nextstrain](https://nextstrain.org/) build for tuberculosis, visible at [nextstrain.org/tb](https://nextstrain.org/tb).
+The results of running this workflow are publicly visible at nextstrain.org/tb
 
-The build encompasses preparing data for analysis, doing quality control, performing analyses, and saving the results in a format suitable for visualization (with [auspice][]). 
-This involves running components of Nextstrain such as [augur][] and [auspice][].
 
-All tuberculosis-specific steps and functionality for the Nextstrain pipeline should be housed in this repository.
 
-## Tutorial
+## Installation
+Follow the [standard installation instructions](https://docs.nextstrain.org/en/latest/install.html) for Nextstrain's suite of software tools.
 
-There is a [tutorial](https://nextstrain.org/docs/getting-started/tb-tutorial) on how to run this build.
+## Usage:
 
-## Usage
+> NOTE: Running this workflow will most likely require more compute resources than what is available on your local computer.
 
-If you're unfamiliar with Nextstrain builds, you may want to follow our
-[quickstart guide][] first and then come back here.
+```
+nextstrain build --image ghcr.io/nextstrain/tb-docker-test:16923552818 .
+```
```

Comment:
We should store the dockerfile and any relevant build documentation / actions in this repo. (If there's compelling reasons not to do this then we should at least clearly document where it is & how it is built in the README.)

`tb-docker-test:16923552818` also implies that this hasn't been finalized, so it'd be nice to make a more official looking tag that is paired with the code in this PR.

Reply:
Yes, we should move all the files from [nextstrain/tb-docker-test](https://github.com/nextstrain/tb-docker-test) into this repo. Everything except for `.github/workflows/docker.yml` can go in its own top-level `docker` folder to keep things organized.

Reply:
This should be a separate PR that is merged first so that the GitHub Actions workflow can run and produce a `ghcr.io/nextstrain/tb:XXXXXX` image for usage in this PR.

@kimandrews if you haven't started on this yet, I'd be happy to do it – I know there's a lot of other things on your TB plate already.